homeassistant/components/sensor/hddtemp.py

```diff
@@ -30,6 +32,7 @@
 SCAN_INTERVAL = timedelta(minutes=1)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_DISKS): vol.All(cv.ensure_list, [cv.string]),
```

Comment:
This will result in a breaking change. Have you thought about a way to make it optional and if not set run with the first disk as default?

Reply:
Thanks for pointing that out. I'll take a look at how to implement that.